plugin-manifest-gen.html

```diff
@@ -0,0 +1,374 @@
+
+            <div class="form-section">
+                <h3>Descriptions:</h3>
+
+                <div class="hint">A short description. Ideally less than 150 characters, must be less than 200.</div>
+                <textarea name="short-description" id="short-description" type="textbox"
+                    placeholder="High-DPI graphics for Endless Sky." rows="1"></textarea>
+                <div class="status" id="short-description-status">0/200</div>
+
+                <div class="hint">A description of arbitrary length.</div>
+                <textarea name="description" id="description" type="textbox" rows="2"
+                    placeholder="This plugin contains double resolution graphics, which are only used if you have a high-dpi monitor or if you set the zoom level above 100%. This will roughly double the amount of memory used by the game."></textarea>
+            </div>
+
+            <div class="form-section">
+                <h3>Download URLs:</h3>
+
+                <div class="hint">A URL to a direct .zip download. It's important that this URL be "versioned", i.e.
+                    contains your version, so it may only point to a download for this release and no other.</div>
+                <input name="download-url" id="download-url" type="text"
+                    placeholder="https://github.com/endless-sky/endless-sky-high-dpi/archive/refs/tags/v0.9.14.zip">
+                <div class="status" id="download-url-status"></div>
+
+                <br>
+                <div class="hint">A URL to your plugin's icon, if any. Make sure this URL is versioned, just like the
+                    one above.</div>
+                <input name="icon-url" id="icon-url" type="text"
+                    placeholder="https://github.com/endless-sky/endless-sky-high-dpi/raw/v0.9.14/icon.png">
+                <div class="status" id="icon-url-status"></div>
+            </div>
+
+
+        </form>
+    </div>
+    <div class="splitter-half" id="splitter-right">
+        <div class="splitter-inner">
+            <h3>Output:</h3>
+            <pre>
+                <code id="output" class="language-yaml"></code>
+            </pre>
+            <div id="copy-output-container">
+                <button id="copy-output">Copy to Clipboard</button>
+            </div>
+        </div>
+    </div>
+</body>
+<script>
+    document.querySelector("#copy-output").addEventListener("click", event => {
+        let text = document.querySelector("#output").textContent;
+        navigator.clipboard.writeText(text);
+    });
+
+    fetch('https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json')
+        .then(response => {
+            if (response.status == 200) {
+                return response.json();
+            }
+            else {
+                return { "licenses": [] };
+            }
+        })
+        .then(json => json.licenses.filter(l => !l.isDeprecatedLicenseId))
+        .then(list => licenses = list)

```

Comment:
No global variables. Use let or const to declare it.

Reply:
I added a `let license = [];`. Does that fix it? I never wrote JS.